ocp4/profiles/e8.profile

```diff
@@ -0,0 +1,141 @@
+documentation_complete: true
+
+title: 'Australian Cyber Security Centre (ACSC) Essential Eight'
+
+description: |-
+    This profile contains configuration checks for Red Hat Enterprise Linux CoreOS
+    that align to the Australian Cyber Security Centre (ACSC) Essential Eight.
+
+    A copy of the Essential Eight in Linux Environments guide can be found at the
+    ACSC website:
+
+    https://www.cyber.gov.au/publications/essential-eight-in-linux-environments
+
+selections:
+
+  ### Remove obsolete packages
+  #- package_talk_removed
+  #- package_talk-server_removed
+  #- package_xinetd_removed
+  #- service_xinetd_disabled
+  #- package_ypbind_removed
+  #- package_telnet_removed
+  #- service_telnet_disabled
+  #- package_telnet-server_removed
+  #- package_rsh_removed
+  #- package_rsh-server_removed
+  #- service_zebra_disabled
+  #- package_quagga_removed
+  #- service_avahi-daemon_disabled
+  #- package_squid_removed
+  #- service_squid_disabled
+
+  ### Software update
+  #- ensure_redhat_gpgkey_installed
+  #- ensure_gpgcheck_never_disabled
+  #- ensure_gpgcheck_local_packages
+  #- ensure_gpgcheck_globally_activated
+  #- security_patches_up_to_date
+  #- dnf-automatic_security_updates_only
+
+  ### System security settings
+  - sysctl_kernel_randomize_va_space
+#  - sysctl_kernel_exec_shield
+  - sysctl_kernel_kptr_restrict
+  - sysctl_kernel_dmesg_restrict
+  - sysctl_kernel_kexec_load_disabled
+  - sysctl_kernel_yama_ptrace_scope
+  - sysctl_kernel_unprivileged_bpf_disabled
+  - sysctl_net_core_bpf_jit_harden
+
+  ### SELinux
+  - var_selinux_state=enforcing
+  - selinux_state
+  - var_selinux_policy_name=targeted
+  - selinux_policytype
+
+  ### Filesystem integrity
+  #- rpm_verify_hashes
+  #- rpm_verify_permissions
+  #- rpm_verify_ownership
+  #- file_permissions_unauthorized_sgid
```

Comment:
No checks for these on OCP4 platform. May be worth adding ocp platform to the rule.yaml definitions for them